data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+
+```<a href="https://piwik.pro">Piwik PRO</a>```
+
+Confguring which domains are outlinks
+"""""""""""""""""""""""""""""""""""""
+
+If you want to track links on your subdomains as a regular PageViews, for example whan your main page is `piwik.pro` and you want to track views of `blog.piwik.pro` as a regular PageViews not outlinks, you have to confgure tracker to recognize this additional domain. You can do it in two ways.
+
+You can configure it in website settings section of the Administration panel. Go to the Administration > Websites & apps > Settings > General settings > URLs. Add all the domains that should not be treated as outlinks.
+
+[Here is the screenshoot of Administration panel]
+
+You can use :ref:`setDomains()<ignoring-alias-domains>` function of JS tracker.
+
+```
+_paq(['setDomains', ["blog.piwik.pro", "piwik.pro", "*.other-domain.pro"]]);
+
+_paq.push(['trackPageView']);
+```
+
+.. note::
+  This will overwrite URLs configured in Administration panel, so use it wisely.
+
+Marking links as outlinks in HTML code
+""""""""""""""""""""""""""""""""""""""
+
+Similar as downloads, links can be set to be treated as outlinks manually, but only with CSS classes, you cannot use a HTML attribute.
+
+You can use one of default CSS classes: `piwik_link` or `piwik-link`. eg. 
+
+```
+<a href='https://piwik.pro' class="piwik-link">Piwik PRO</a>
+```
+
+Or you can define your custom CSS classes for outlinks with ours :ref:`Javascript Tracker API<force-tracking-using-css-class>`. 
+
+```
+// now all clicks on links with the css class "external" will be counted as outlinks
+
+// you can also pass an array of strings
+_paq.push(['setLinkClasses', "custom-link-class"]);
+
+_paq.push(['trackPageView']);
+```
+
+or a list of classes
+
+```
+_paq.push(['setLinkClasses', ["custom-link-class", "other-link-class"]]);
+
+_paq.push(['trackPageView']);
+```
+
+and in HTML code
+
+```
+<a href='https://piwik.pro' class="custom-link-class">Piwik PRO</a>
+```
+
+Marking links with inline Javascript
+""""""""""""""""""""""""""""""""""""
+
+Alternatively you can use an inline javascript and onclick attribute to track any link as an outlink.
+
+```
+<a href="mailto:support@piwik.pro" target="_blank" onClick="_paq.push(['trackLink', 'https://piwik.pro/support-contact-form', 'link']);">Write us a message.</a>
+```
+
+
+Other related  abilities
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Changing delay for link tracking
+""""""""""""""""""""""""""""""""
+All link tracking uses a slight delay of click execution, so the browser won't exit the page before a click is tracked. The default value of such delay is 500ms, but you can modify it as you wish. You have to remember that if you set this value too low, it might be not enough to track the click, if you set it too high, the browser might ignore the delay.
+
+```
+_paq.push(['setLinkTrackingTimer', 300]); // 300 milliseconds
+
+_paq.push(['trackPageView']);
```

Comment:
Is the track page view nescessary?

Reply:
https://github.com/PiwikPRO/PPMS-PublicDocs/pull/629/files#r705232674